tests/utils/command.go

```diff
@@ -58,13 +60,36 @@ func RegisterPingTest() {
 	})
 }
 
-// RegisterNodeRun registers a before suite that starts an AvalancheGo process to use for the e2e tests
-// and an after suite that stops the AvalancheGo process
+// At boot time subnets are created, one for each test suite. This global
+// variable has all the subnets IDs that can be used.
+//
+// One process creates the AvalancheGo node and all the subnets, and these
+// subnets IDs are passed to all other processes and stored in this global
+// variable
+var BlockchainIDs map[string]string
+
+// Timeout to boot the AvalancheGo node
+var bootAvalancheNodeTimeout = 5 * time.Minute
+
+// Timeout for the health API to check the AvalancheGo is ready
+var healthCheckTimeout = 5 * time.Second
```

Comment:
style nit: we should define top-level vars in a block where possible, also seems the latter 2 could be constants